apps/fluent-tester/src/FluentTester/TestComponents/Avatar/AvatarTest.tsx

```diff
@@ -31,7 +31,7 @@ const avatar: React.FunctionComponent<{}> = () => {
 
 const stylizedAvatar: React.FunctionComponent<{}> = () => {
   const CustomizedAvatar = Avatar.customize({
-    size: 'large',
+    size: 'xxLarge',
```

Comment:
nit: it would be nice to show all the sizes in test app.

Reply:
Yes, I think I'll update the test to do that.